mobile/lib/modules/home/views/home_page.dart

```diff
@@ -78,6 +85,18 @@ class HomePage extends HookConsumerWidget {
 
           lastMonth = currentMonth;
         });
+
```

Comment:
Let's remove dead code

Reply:
This was for the Hive box, just commented out for now.